firmware/port.c

```diff
@@ -321,6 +324,30 @@ ExecStatus port_begin_cmd(PortData *p) {
             port_send_status(p, REPLY_DATA);
             return EXEC_CONTINUE;
 
+        case CMD_WAIT: {
+            // Parse the microsecond delay arguments
+            u32 us_delay = (p->arg[3]) |
+                           (p->arg[2] << 8) |
+                           (p->arg[1] << 16) |
+                           (p->arg[0] << 24);
+
+            // 1s/ 48e6 ticks * 1e6 us / s = 48 ticks/us
+            // TODO: I think we also have to incorporate the 256 div prescalar
+            u32 ticks = 48 * us_delay;
+
+            // If the delay is 0, just return now
+            if (ticks == 0) {
+              return EXEC_DONE;
+            }
+
+            // Enable delays on this TCC channel
+            tcc_delay_enable(p->port->delay_id);
```

Comment:
@kevinmehall is there any reason we can't re-use the TC_BOOT TC for port B? Then they could both use `TC`s instead of having to implement switching logic for TCs and TCCs.